docs/basics/part-7-using-remote-runners.md

```diff
@@ -0,0 +1,88 @@
+Earthly has the ability to run builds both locally and remotely. If you followed the standard [installation instructions](https://earthly.dev/get-earthly), then you most likely have only run local builds so far. In this section, we will explore how to use remote runners to perform builds on remote machines.
+
+## Remote runners
+
+Earthly is able to use remote runners for performing builds on remote machines. When Earthly uses a remote runner, the inputs of the build are picked up from the local environment, then the execution takes place remotely, including any pushes (`RUN --push` commands, and `SAVE IMAGE --push` commands), but any local outputs are sent back to the local environment. All this takes place while your local Earthly process still provides the logs of the build in real time locally.
+
+Remote runners are especially useful in a few specific circumstances:
+
+* You want to reuse cache between CI runs to dramatically speed up builds (more on this in part 8).
+* You want to share compute and cache with colleagues and/or with the CI.
+* You have a build that requires a lot of resources, and you want to run it on a machine with more resources than your local machine.
+* You have a build that requires to run on a specific CPU architecture natively.
+* You have a slow internet connection.
+
+There are two types of remote runners:
+
+* Remote Buildkit (self-hosted)
+* Earthly Satellites
+
+### Using a remote Buildkit
+
+A common way to use remote runners is to deploy your own instance of Buildkit and have Earthly connect to it. The [remote Buildkit page](../ci-integration/remote-buildkit.md) has more information on how to set this up.
+
+Once the remote Buildkit is up and running, you may use it from Earthly by setting the configuration option `buildkit_host` to the address of the remote Buildkit. For example, if the remote Buildkit is running on `earthly-remote-buildkit.example.com`, you may set the configuration option
```

Comment:
Is it worth mentioning the options to this with the cli argument?

Reply:
What do you mean specifically?

Reply:
The option to set the remote host with `--buildkit-host` instead of setting it in the configuration file.

Reply:
Ah yes - I see.

https://github.com/earthly/earthly/pull/2900